### ISSUE TITLE
Plasma Large Turbine energy generation incorrect by factor of turbine efficiency

### ISSUE THREAD

**Issue by loving2** [GregTechCEu/GregTechCEu-1.19#363]
### GregTech CEu Version

gtceu-forge-1.20.1-1.0.11.b

### Recipe Viewer Installed

JEI

### Environment

Tested on both

### Cross-Mod Interaction

Unsure

### Expected Behavior

Expected a plasma turbine to produce the amount of energy the controller block claims it is producing

### Actual Behavior

The amount of energy produced is very nearly a factor of turbine efficiency less

### Steps to Reproduce

I set up a plasma large turbine using an HSS-G Turbine, ZPM Rotor Holder, and ZPM 4a Dynamo Hatch
I attached a Powah Ender Cell with 20B RF storage to the Dynamo Hatch for energy storage
I utilized Integrated Dynamics Logic Programmer, Block Reader, Variable Store, Delayer, and Display Panel to read the NBT of the Ender Cell and took the difference of energy stored over 10 seconds. Dividing by 200 to get it to RF/t, then 4 to EU/t, resulted in a value of ~108k EU/t rather than the 183.5k EU/t the controller claims to make.

The EU/t can also be seen via Jade and is perhaps the easier method of determining it, but I wanted to be certain

![EU per tick](https://github.com/GregTechCEu/GregTech-Modern/assets/26778483/8e6a9ff8-ff6a-4ec7-b1a1-c84c1e98ba56)
![Jade](https://github.com/GregTechCEu/GregTech-Modern/assets/26778483/82bb5dfd-349b-4848-a798-efc4a5531cda)
![Turbine Controller Interface](https://github.com/GregTechCEu/GregTech-Modern/assets/26778483/e00b9624-8fed-4a79-9d0a-784a7b88f3a8)

**Comment by loving2** [comment]
I've looked at the code and believe I have found where the [EU/t calculation is](https://github.com/GregTechCEu/GregTech-Modern/blob/1.19.2/common/src/main/java/com/gregtechceu/gtceu/common/machine/multiblock/generator/LargeTurbineMachine.java#L94)

My understanding of this code is that I am punished for having a turbine efficiency greater than 1. Why is that the case? What should efficiency do? My assumption was that it would reduce fuel consumption, not energy production.

The Energy Output in the GUI matches the turbineMaxVoltage variable value, why does the eut variable not match it at max speed?
